index.html

```diff
@@ -36,26 +42,24 @@ <h4 class="headline">LET’S CONNECT</h4>
       <li><a href="#"><img src="images/Vector(1).svg" alt="gitbubicon" /></a></li>
       <li><a href="#"><img src="images/Vector.svg" alt="thumb" /></a></li>
     </ul>
-  </section>
+  </div>
+</section>
 
-
-
-
-
-
-  
-  <section>
-    <div class="cards-container">
-      <ul class="cards-list flex-center-culomn">
-        <li class="card-item flex-center-culomn">
+<section>
+  <div class="cards-container">
+    <ul class="cards-list flex-center-culomn">
+      <li class="card-item flex-center-culomn">
+        <div class="img-container">
           <img class="card-image" src="images/SnapshootPortfolio.jpg" alt="Portfolio Portfolio">
+        </div>
```

Comment:
- [ ] The background color of cards should be lighter `#F4F5F7`, in your case it is slightly dark. 
- [ ] Please set the border of cards to `#DFE1E6` to make it lighter as in Figma: 

Your design  | Figma design
------------- | -------------
<img width="765" alt="Screenshot 2022-04-01 at 12 25 01" src="https://user-images.githubusercontent.com/77038610/161225249-46990e5d-2223-4b2c-aff9-42fcfbf283ec.png"> | <img width="887" alt="Screenshot 2022-04-01 at 12 24 43" src="https://user-images.githubusercontent.com/77038610/161225204-bcdbdb55-bba9-4907-89a9-bfcfdd758cbe.png">

- [ ] Please add a little more padding inside the language tags and set the color to `#6070FF`: 

Your design  | Figma design
------------- | -------------
<img width="154" alt="Screenshot 2022-04-01 at 12 26 36" src="https://user-images.githubusercontent.com/77038610/161225545-791beeb4-8b96-43fc-9ed8-0c43a5ae9e81.png">  | <img width="158" alt="Screenshot 2022-04-01 at 12 26 14" src="https://user-images.githubusercontent.com/77038610/161225469-496365ad-38f0-4be7-8d49-e9be1e7dd9b8.png">